modin/pandas/test/test_general.py

```diff
@@ -27,11 +27,23 @@
     test_data_values,
     test_data_keys,
     df_equals,
+    assert_index_equal,
     sort_index_for_equal_values,
     eval_general,
+    default_to_pandas_ignore_string,
+    bool_arg_values,
+    bool_arg_keys,
 )
 
 
+# Our configuration in pytest.ini requires that we explicitly catch all
+# instances of defaulting to pandas, but some test modules, like this one,
+# have too many such instances.
+# TODO(https://github.com/modin-project/modin/issues/3655): catch all instances
+# of defaulting to pandas.
+pytestmark = pytest.mark.filterwarnings(default_to_pandas_ignore_string)
```

Comment:
Why not just use `warns_that_defaulting_to_pandas`?

Reply:
fixed